src/app/components/CourseChoiceButton/CourseChoiceButton.module.css

```diff
@@ -1,8 +1,7 @@
 .button {
   color: var(--contrast-text-color);
   background: var(--primary-color);
-  font-size: 1.25em;
-  font-weight: bold;
+  font-size: 1em;
```

Comment:
Is this necessary? does it change the standard font size?

Reply:
Thank you. Changed it to a link which changed the styling completely.